### PR TITLE
SHOT-3486 - Updated display group name in scene_browser.py

### PR DESCRIPTION
**Pull request by rob-aitchison**
Updated .pre-commit-config.yaml
pre-commit updated tk-nuke_scene_operations.py
Updated .gitignore for .idea and venv folders

Just finishing up the Group name from this previous PR: https://github.com/shotgunsoftware/tk-multi-breakdown/pull/17

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -14,11 +14,13 @@ exclude: "ui\/.*py$"
 # List of super useful formatters.
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.4.0
+    rev: v3.2.0
```
Comment (pscadding) [inline .pre-commit-config.yaml:17]: I'm not personally against updating this, I just wondered what drove you to update it?

**Review by pscadding — CHANGES_REQUESTED** [review]
I can see you've added `.idea` to the gitignore, but the `.idea` files have also been added.
Also black would like you to make a change.

The actual functional change looks great though!

**Review by pscadding — APPROVED** [review]
Looks great!

**Comment by rob-aitchison** [comment]
Hey @pscadding thanks for the quick approval!

The hound change was bizarre and we should keep our eyes peeled for this. I locally ran the pre-commit hooks and it modified the tk-nuke_scene_operations.py file. I then did the commit and push and it sent the previous formatting which triggered the error. Either I was seeing things late in the day or there is something in my Pycharm cache or a mismatch between config files? A Windows anomaly?

Just flagging for awareness - cheers!

File: .pre-commit-config.yaml (hunk 1 above)
Comment (jfboismenu) [reply .pre-commit-config.yaml:17]: 3.2.0 fixes an issue on Windows regarding the shebang pre-commit hook.

**Pull request merged**